assets/css/components/OutputPane.scss

```diff
@@ -34,3 +34,18 @@ iframe + .outputpane-box {
   right:0;
   bottom:0;
 }
+
+.overlapping .outputpane.focus .outputpane-box {
+  z-index: 2;
+  box-shadow: -2px 2px 5px rgba(0,0,0,0.3);
+}
+
+.overlapping .outputpane:not(.focus):after {
```

Comment:
:after is unusual and I don't think I understand what this CSS rule is doing. Can you add a comment?